libcxx/test/libcxx/atomics/atomics.types.generic/trivially_copyable.verify.cpp

```diff
@@ -0,0 +1,28 @@
+//===----------------------------------------------------------------------===//
```

Comment:
I don't know why you thought this test file was misplaced. IIUC it's testing a standard requirement, so it seems reasonable to me to place it in `libcxx/test/std` subdirectory.

Reply:
Because, as I understand it, that test is checking for a particular error message from a failed static assertion and that message is libcxx-specific, not something that is standardized.
Let me know if I am wrong and I need to undo the move.

Reply:
It is true that these checks are libc++-specific in the end. However, they test standards-mandated behaviour. I'm pretty sure nobody is running `.verify.cpp` files on anything other than libc++, but I think a generic "this test fails in some way" is completely valid. That we test for the specific error-messages via a special compiler mode doesn't change that IMO, since they're just comments.

Reply:
I agree here, this is technically not specific to libc++ even though we don't implement proper checking for any other stdlib right now. Let's move it back to `libcxx/test/std`.

Reply:
Done